src/migrations/create.rs

```diff
@@ -788,11 +788,11 @@ pub async fn normal_migration(cli: &mut Connection, ctx: &Context,
 }
 
 fn add_newline_after_comment(value: &mut String) -> Result<(), anyhow::Error> {
-    let last_token = TokenStream::new(value).last()
+    let last_token = Tokenizer::new(value).last()
         .ok_or_else(|| bug::error("input should not be empty"))?
         .map_err(|e| bug::error(
-            format!("tokenizer failed on reparsing: {}", e)))?;
-    let token_end = last_token.end.offset as usize;
+            format!("tokenizer failed on reparsing: {}", e.message)))?;
```

Comment:
What's wrong with `fmt::Display` of the error?

It's also probably should use `{:#}` in error format string, as that usually gives more detailed info on error (by a convention introduced by `anyhow`).

Reply:
Oh, that's just a biproduct of changing errors from `combine::easy::Error` to our `edgeql_parser::Error`. I can implement Display for our error as well.